doc/userGuide.md

```diff
@@ -175,9 +175,9 @@ for more information.
 
 ```bash
 ask new
-> choose a skill hosting method.
 > select NodeJS
-> select Hello World (with Controls)
+> choose a skill hosting method, except "Alexa-hosted skills" as it doesn't support custom template selection
```

Comment:
You will also need to make changes in the paragraph just before this snippet. Also, I think it is better to mention that `Alexa-hosted skills` custom template selection is not supported at the moment?

Reply:
Good call, done